src/main/java/sirius/kernel/cache/CacheManager.java

```diff
@@ -57,12 +57,12 @@ private CacheManager() {
     }
 
     /**
-     * Returns a list of all known caches
+     * Returns a sorted list of all known caches
      *
-     * @return a list of all caches created so far
+     * @return a sorted list of all caches created so far
      */
     public static List<ManagedCache<?, ?>> getCaches() {
-        return new ArrayList<>(caches.values());
+        return caches.values().stream().sorted().collect(Collectors.toList());
     }
```

Comment:
```suggestion
        return caches.values().stream().sorted(Comparator.comparing(Cache::getName)).collect(Collectors.toList());
    }
```
würde auch gehen, dann spart man sich den comparator unten